layouts/default.vue

```diff
@@ -1,26 +1,24 @@
 <template>
 	<div id="defaultContainer">
 		<Header />
-		<div class="content-wrapper">
-			<Aside />
-			<main>
-				<nuxt />
-			</main>
-		</div>
+		<nuxt />
 		<Footer />
 	</div>
 </template>
 
 <script>
 import Header from "~/components/Header";
-import Aside from "~/components/Aside";
 import Footer from "~/components/Footer";
 export default {
 	components: {
 		Header,
-		Aside,
 		Footer
 	},
+	data () {
```

Comment:
This `data` block can be removed as `<Aside>` has been moved out of this vue file.